src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/SpanSelectorLabel.tsx

```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/css';
+import { Button, Tooltip, useStyles2 } from '@grafana/ui';
+import React from 'react';
+
+export function SpanSelectorLabel({ spanSelector }: { spanSelector: string }) {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.container}>
+      <Tooltip content={`You have added a span selector to the flamegraph query (${spanSelector}).`} placement="top">
+        <span>Span selector added</span>
+      </Tooltip>
+      <Button
+        size="md"
+        fill="text"
+        variant="secondary"
+        icon="times"
+        tooltip={`Remove span selector from query`}
+        tooltipPlacement="top"
+        onClick={() => {}}
```

Comment:
Looks like we needed to use a variable for state management. But I will need to utilise your expertise here 
@ifrost for the best way to remove the spanSelector when this button is clicked? (not sure what way to go about this)

Reply:
This can be done with events. The lower-level component dispatches an event ([example](https://github.com/grafana/explore-profiles/blob/39e550a169323f53e8b0a4f9481f34009ea7e8f3/src/pages/ProfilesExplorerView/components/SceneExploreDiffFlameGraph/components/SceneDiffFlameGraph/SceneDiffFlameGraph.tsx#L130)) and the root component which control variables can listen to the event and change variable accordingly ([example](https://github.com/grafana/explore-profiles/blob/1c7b6c3f537b9ff0bc9060bac5f0e2c240ba00c8/src/pages/ProfilesExplorerView/components/SceneProfilesExplorer/SceneProfilesExplorer.tsx#L231)).